plugins/org/src/components/MyGroupsSidebarItem/MyGroupsSidebarItem.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  SidebarItem,
+  SidebarSubmenu,
+  SidebarSubmenuItem,
+} from '@backstage/core-components';
+import {
+  IconComponent,
+  identityApiRef,
+  useApi,
+  useRouteRef,
+} from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  catalogApiRef,
+  CatalogApi,
+  entityRouteRef,
+} from '@backstage/plugin-catalog-react';
+import { getCompoundEntityRef } from '@backstage/catalog-model';
+
+/**
+ * MyGroupsSidebarItem can be added to your sidebar providing quick access to groups the logged in user is a member of
+ *
+ * @public
+ */
+export const MyGroupsSidebarItem = ({
+  props,
+}: {
+  props: { singularTitle: string; pluralTitle: string; icon: IconComponent };
+}) => {
+  const { singularTitle, pluralTitle, icon } = props;
+  const identityApi = useApi(identityApiRef);
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const catalogEntityRoute = useRouteRef(entityRouteRef);
+
+  const { value: groups } = useAsync(async () => {
+    const profile = await identityApi.getBackstageIdentity();
+
+    const response = await catalogApi.getEntities({
+      filter: [{ kind: 'group', 'relations.hasMember': profile.userEntityRef }],
+      fields: ['metadata', 'kind'],
+    });
+
+    return response.items;
+  }, []);
+
+  // Not a member of any groups
+  if (!groups?.length) {
+    return <></>;
```

Comment:
I would `return null` here, or is there a reason for rendering an empty fragment?

Reply:
Sure, I have no real reason for the empty fragment other than I think in another PR it was suggested to me so that's why I did it here.

Reply:
Ok, changed this to `return null`